dbtmetabase/models/exceptions.py

```diff
@@ -6,6 +6,10 @@ class NoDbtSchemaSupplied(Exception):
     """Thrown when using folder parser without supplying a schema"""
 
 
+class NoMetabaseCredentialsOrSessionIdSupplied(Exception):
```

Comment:
Session ID is a type of credential too, so should just be `NoMetabaseCredentialsSupplied`

Reply:
updated